src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -546,7 +546,19 @@ static void Main()
     }
 }";
             var comp = CreateCompilation(source);
-            comp.VerifyEmitDiagnostics();
```

Comment:
Should we ensure that the non-'ref' version of the assignment is permitted, since it presumably refers to a real variable?